db/seeds/development/1_people.rb

```diff
@@ -28,11 +28,9 @@ def seed_demo_person(email, group, role_type)
 
 puzzlers = ['Pascal Zumkehr',
             'Pascal Simon',
-            'Pierre Fritsch',
             'Andreas Maierhofer',
-            'Andre Kunz',
             'Matthias Viehweger',
-            'Roland Studer']
+            'Olivier Brian']
```

Comment:
```suggestion
            'Olivier Brian',
            'Carlo Beltrame']
```

Ich schlage vor, dass wir Carlo hier auch gleich hinzufügen. :-)